### DISCUSSION TITLE
Why is "Cli" embedded in Spectre.Console?

### DISCUSSION THREAD

**Discussion by olivier-spinelli** [spectreconsole/spectre.console#328] [Q&A] [answered]
From the currently active develop branch, we can see that Spectre.Console comes with a Command Line parsing/command model/executor component.
This Cli part is interesting, however, isn't it confusing?
There's a lot of alternative  for a commmand-line-parser-and-execution micro framework, sometimes very light (and this is welcome).

Why is "Cli" not in a **Spectre.Console.Cli** package that depends on a minimalist **Spectre.Console**?

**Comment by patriksvensson** [comment, marked as answer]
@olivier-spinelli For two reasons; It depends on Spectre.Console for its internals, and it's something that we personally use a lot and want to include in our little "console package". Since it's included in a separate namespace it doesn't really affect anything else.

To be honest, I personally don't see it's as "confusing", but I guess that depends on what your picture of "Spectre.Console" is.
